content/neuvector-prime/azure/contents.lr

```diff
@@ -190,53 +199,46 @@ Billing will be available in Azure Portal billing
 Home > Cost Management <subscription> | Cost analysis
 
 
+## Enable Auto Upgrade minor version
 
-## Upgrading a NeuVector PAYG Cluster
+By default the Kubernetes Service Extension "Auto Upgrade minor version" will be in a Disabled state.
```

Comment:
I would include something above this that explains why the customer might want to turn this on... Maybe "Upgrading your NeuVector offering requires the use of the Azure automatic upgrade functionality. This will allow NeuVector to regularly update its database of threat definitions."